compiler/resolution/fixupExports.cpp

```diff
@@ -181,30 +183,34 @@ static bool validateFormalIntent(FnSymbol* fn, ArgSymbol* as) {
   // TODO: If we ever add more types to these fixup routines, we really ought
   // to put these conditions in tables.
   //
-  if (t == dtString) {
+  if (t == dtString || t == dtStringC || t == dtExternalArray) {
     IntentTag tag = as->intent;
 
-    if (fMultiLocaleInterop || strcmp(CHPL_COMM, "none")) {
+    bool multiloc = fMultiLocaleInterop || strcmp(CHPL_COMM, "none");
+
+    if (multiloc || willBePythonized(fn)) {
       // TODO: After resolution, have abstract intents been normalized?
       if (tag != INTENT_IN &&
           tag != INTENT_CONST_IN) {
+        const char* libdesc = multiloc ? "multilocale" : "python";
```

Comment:
What about a _multilocale_ python library? xD

Reply:
I've decided that's a bug fix :P